deafrica_waterbodies/plugins/ocean_filtering_using_goas.py

```diff
@@ -0,0 +1,92 @@
+"""
+Ocean filtering using Marine Regions Global Oceans and Seas v01
+"""
+import os
+
+import geopandas as gpd
+import numpy as np
+import skimage.morphology
+import xarray as xr
+from deafrica_tools.spatial import xr_rasterize
+
+# File extensions to recognise as Parquet files.
+PARQUET_EXTENSIONS = {".pq", ".parquet"}
+
+buffer_pixels = 500 / 30
```

Comment:
Rather than hard-coding this, would it be useful to have the number of metres to buffer as an input argument to erode_land_sea_mask, and then inside that function, convert from the number of metres to the number of pixels using the actual resolution of the loaded mask?